Frontend/ui-customer/src/Components/IndividualRideCard/View.purs

```diff
@@ -73,21 +75,56 @@ cardView push state =
   , cornerRadius 8.0
   , stroke $ "1,"<>Color.grey900
   , background Color.white900
-  ][  rideDetails push state
-    , separator
-    , sourceAndDestination push
-    , separator
-    , viewDetailsAndRepeatRide push state
+  ][  zoneView state
+    , linearLayout
+      [ height WRAP_CONTENT
+      , width MATCH_PARENT
+      , padding $ Padding 16 0 16 0
+      , cornerRadii $ Corners 8.0 true true true true
+      , orientation VERTICAL
+      , background Color.white900
+      ][  rideDetails push state
+        , separator
+        , sourceAndDestination push
+        , separator
+        , viewDetailsAndRepeatRide push state
+      ]
    ]
 
-rideDetails :: forall w. (Screen.Action -> Effect Unit) -> IndividualRideCardState ->  PrestoDOM (Effect Unit) w 
-rideDetails push state = 
+zoneView :: forall w. IndividualRideCardState ->  PrestoDOM (Effect Unit) w
+zoneView state =
+  linearLayout
+  [ width MATCH_PARENT
+  , height WRAP_CONTENT
+  , background Color.blue800
+  , padding (PaddingVertical 5 5)
+  , orientation HORIZONTAL
+  , gravity CENTER
+  , cornerRadii $ Corners 8.0 true true false false
+  , visibility if state.zoneType == METRO then VISIBLE else GONE
+  ][ imageView
+     [ width (V 15)
+     , height (V 15)
+     , margin (MarginRight 6)
+     , imageWithFallback "ny_ic_metro_white,https://assets.juspay.in/nammayatri/images/common/ny_ic_metro_white.png"
```

Comment:
@ashishsingh101 This s3 url doesn't have any image can you pr in assetsStore

Reply:
sure